.github/scripts/uploadE2eBuildToEmerge.ts

```diff
@@ -78,5 +78,5 @@ $.exec(`bundle exec fastlane run emerge \
   base_sha:${baseSha} \
   branch:${branchName} \
   pr_number:${prNumber || ''} \
-  build_type:e2e
+  tag:e2e
```

Comment:
`build_type` now is named `tag` (see [changelog](https://github.com/EmergeTools/fastlane-plugin-emerge/blob/77e4484aae38b3c0d98a3a22ff9b9206d691e770/CHANGELOG.md#070))